### PR TITLE
Add juliamarkdown support

### PR DESCRIPTION
**Pull request by ZacLN**
NOTE: should follow #161 
Gives LanguageServer.jl control over juliamarkdown files.

### CODE REVIEW

**Comment by ZacLN** [comment]
@davidanthoff, JMD support is ready! Similarly to before this needs to be approved/merged in conjunction w/ [this](https://github.com/JuliaEditorSupport/LanguageServer.jl/pull/59)

File: package.json [hunk 1]
```diff
@@ -107,6 +108,10 @@
             {
                 "command": "language-julia.weave-save",
                 "title": "julia Weave: Save to file..."
+            },
+            {
+                "command": "language-julia.lint-package",
+                "title": "julia: Lint Package"
```
Comment (davidanthoff) [inline package.json]: Hm, this shouldn't show up here, right?

File: src/extension.ts [hunk 2]
```diff
@@ -79,6 +79,9 @@ export function activate(context: vscode.ExtensionContext) {
 
     let applytextedit = vscode.commands.registerCommand('language-julia.applytextedit', applyTextEdit);
     context.subscriptions.push(applytextedit);
+
+    let lintpkg = vscode.commands.registerCommand('language-julia.lint-package', lintPackage);
+    context.subscriptions.push(lintpkg);
```
Comment (davidanthoff) [inline src/extension.ts]: Same here, wasn't this stuff in the previous PR?

File: src/extension.ts [hunk 3]
```diff
@@ -524,3 +527,7 @@ export function applyTextEdit(we) {
         vscode.workspace.applyEdit(wse)
     }
 }
+
+export function lintPackage() {
+    languageClient.sendRequest("julia/lint-package")
+}
```
Comment (davidanthoff) [inline src/extension.ts]: And this one as well.

Comment (ZacLN) [inline src/extension.ts]: I don't understand this, maybe push an empty commit and see if that clears this up

**Pull request closed without merging**